db/redis.js

```diff
@@ -70,11 +72,16 @@ async function getLatestLocationStatesData(locationid){
     return singleitem[0];
 }
 
+function disconnect() {
```

Comment:
Needed to add this so that I could turn the Redis server off after the tests were finished.